### ISSUE TITLE
EPIC: Review accessibility guidance

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual#1948]
Our analytics suggest that the accessibility guidance is not well used. Some of our user research 2024 suggests that some professions (e.g. product and delivery, devs and designers) do not refer to the profession sections.

It would be worth reviewing the guidance - and how best to present accessibility requirements and best practice across:
- the design system
- the accessibility guidance
- the accessibility checklist
Also taking into account cross-gov work on accessibility guidance and resources.

**Comment by anandamaryon1** [comment]
Also from the 'WCAG 2.2 user research 2024', of the users that had used the per-profession accessibility guidance, they had accessed it when starting their role, as part of onboarding. They had not come back to it since. 

Two thoughts:
* does this suggest that the accessibility content in general should be more integrated into the design system and content content guide?
* is there a distinction between content that is used routinely as a reference and content that is more akin to training or onboarding? Should these be treated differently in the Service Manual?

**Comment by sarawilcox** [comment]
Also, DigInclusion's report on the accessibility guidance identified that we could add more info on all WCAG 2.1 criteria. For now, we have decided not to do that and only to highlight the 6 new criteria - in addition to design system updates. For now, we do not know if our users will want more detail about all WCAG 2.1 criteria (these are covered in the accessibility checklist) but we suspect this will be too much for them. Worth exploring as part of the review.

**Comment by sarawilcox** [comment]
There are also some accessibility resources on NHSE's CoE SharePoint, including some duplicate content.

**Comment by sarawilcox** [comment]
We will be carry out some user research into people's needs around accessibility guidance, starting June 2024, and how we can better meet those needs in the service manual.
